File: Leetcode/171-Excel-Sheet-Column-Number/ExcelSheetColumnNumber.java
public class ExcelSheetColumnNumber {
    
    public int titleToNumber(String s) {
        if (s == null) {
            return -1;
        }
        if (s.length()<1) {
            return 0;
        }
        int total = 0;
        for (int i=0; i<s.length(); i++) {
            total = total*26;
            total += s.charAt(i)-'A'+1;
        }
        return total;
    }// end of titleToNumber           
    
}

File: Leetcode/171-Excel-Sheet-Column-Number/ExcelSheetColumnNumber.js
/**
 * @param {string} s
 * @return {number}
 */
var titleToNumber = function(s) {
    if (!s) {
        return -1
    }
    let len = s.length
    if (len<1) {
        return 0
    }
    var total = 0;
    var i;
    for (i=0; i<len; i++) {
        total *= 26
        total += s[i].charCodeAt(0) - 'A'.charCodeAt(0) +1
    }
    return total
};
